src/components/bookings/equipmentLists/SelectNumberOfUnitsAndHoursModal.tsx

```diff
@@ -45,6 +50,10 @@ const SelectNumberOfUnitsAndHoursModal: React.FC<Props> = ({
         }
     }, [numberOfUnitsRef, numberOfHoursRef]);
 
+    useEffect(() => {
+        setSelectedPriceId(equipment.prices[0].id);
+    }, [equipment]);
```

Comment:
Varför behövs den här useEffect?

Reply:
Ah tror jag fattar nu.  Det är för att modalen ska uppdatera sitt state varje gång man väljer en ny utrustning va?

Reply:
Hmm, jag minns faktiskt inte. Det är möjligt att du har rätt, men samtidigt tyckte jag det funkade likadant utan. Undrar om det är en kvarleva från att jag ett tag prövade att ladda in equipment från APIt i denna komponent, så den inte var definierad från start.

Reply:
Ja det funkar ju prima att ta bort den så jag gör det